docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,249 @@
+2. **Keep it simple**: The CLI should be all you strictly need to interact with global environments.
+3. **Unsurprising**: Simple commands should behave similar to traditional package managers.
+4. **Human Readable**: Any file created by this feature should be human-readable and modifiable.
+
+## Manifest
+
+The global environments and exposed will be managed by a human-readable manifest.
+This manifest will stick to conventions set by `pixi.toml` where possible.
+Among other things it will be written in the TOML format, be named `pixi-global.toml` and be placed at `~/.pixi/manifests/pixi-global.toml`.
+The motivation for the location is discussed [further below](#multiple-manifests)
+
+```toml title="pixi-global.toml"
+# The name of the environment is `python`
+# It will expose python, python3 and python3.11, but not pip
+[envs.python.dependencies]
+python = "3.11.*"
+pip = "*"
+
+[envs.python.exposed]
+python = "python"
+python3 = "python3"
+"python3.11" = "python3.11"
+
+# The name of the environment is `python_3_10`
+# It will expose python3.10
+[envs.python_3_10.dependencies]
+python = "3.10.*"
+
+[envs.python_3_10.exposed]
+"python3.10" = "python"
+
+```
+
+## CLI
+
+Install one or more packages `PACKAGE` and expose their binaries.
+If `--environment` has been given, all packages will be installed in the same environment.
+If the environment already exists, the command will return with an error.
+`--expose` can be given if `--environment` is given as well or if only a single `PACKAGE` will be installed.
+The syntax for `MAPPING` is `exposed_name=binary_name`, so for example `python3.10=python`.
+
+```
+pixi global install [--expose MAPPING] [--environment ENV] <PACKAGE>...
+```
+
+Remove environments `ENV`.
+```
+pixi global uninstall <ENV>...
+```
+
+Update `PACKAGE` if `--package` is given. If not, all packages in environments `ENV` will be updated.
+If the update leads to binaries being removed, it will offer to remove the mappings.
+If the user declines the update process will stop.
+If the update leads to binaries being added, it will offer for each binary individually to expose it.
+```
+pixi global update [--package PACKAGE] <ENV>...
+```
+
+Add one or more packages `PACKAGE` into an existing environment `ENV`.
+If environment `ENV` does not exist, it will return with an error.
+Without `--expose` no binary will be exposed.
+If you don't mention a spec like `python=3.8.*`, the spec will be unconstrained with `*`
+
+```
+pixi global add --environment ENV [--expose MAPPING] <PACKAGE>...
+```
+
+Remove package `PACKAGE` from environment `ENV`.
+If that was the last package remove the whole environment and print that information in the console.
+If this leads to binaries being removed, it will offer to remove the mappings.
+If the user declines the remove process will stop.
+```
+pixi global remove --environment ENV PACKAGE
+```
+
+Add one or more `MAPPING` for environment `ENV` which describe which binaries are exposed
```

Comment:
I think it would be nice to mention the syntax of mapping for this

Reply:
Done